### PR TITLE
Fixed ActiveDirectoryInteractive authentication timeout issue

### PR DESCRIPTION
**Pull request by lilgreenbird**
Use 10 * loginTimeout timeout value which is consistent with SqlClient behaviour

### CODE REVIEW

File: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java [hunk 1]
```diff
@@ -870,6 +870,8 @@ IdleConnectionResiliency getSessionRecovery() {
         return sessionRecovery;
     }
 
+    static boolean isWindows;
```
Comment (v-zhangw) [inline src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java:873]: What is this for?

**Review by tkyc — APPROVED** [review]

**Review by v-zhangw — APPROVED** [review]

**Comment by tkyc** [comment]
Was just reading the files, and noticed this alot in project. So the convention for null checks is "null == someVar"? And not the other way around? eg. "someVar == null" or "someVar != null"

**Pull request merged**